src/Lykke.Cqrs/Middleware/CommandInterceptorContext.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Threading.Tasks;
+
+namespace Lykke.Cqrs.Middleware
+{
+    internal class CommandInterceptorContext : ICommandInterceptionContext
```

Comment:
I suggest to name this class CommandInterceptionContext and rename current CommandInterceptionContext to the CommandInterceptionContextData or CommandInterceptionCommonContext or CommandInterceptionSharedContext or something like that